common/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("application")
+    id("com.hedera.block.common")
+}
```

Comment:
seems redundant to set `application` plugin when is already part of the "com.hedera.block.common" plugin.
Also, see prior comment between `application` and `java-library`

Reply:
addressed in 51da6a1d654c0919796b6dbdd1691fa239c5e3f0 can you please verify?